db/migrate/20170221093606_update_old_redirect_unpublishings.rb

```diff
@@ -0,0 +1,24 @@
+class UpdateOldRedirectUnpublishings < ActiveRecord::Migration[5.0]
+  def up
+    Unpublishing
+      .where(type: :redirect)
+      .where(redirects: nil)
+      .find_each do |unpublishing|
+        # this magical line works because the 'redirects' getter is overriden
+        # to generate a JSON blob automatically if it is nil within the
+        # database
+        unpublishing.update_attribute(:redirects, unpublishing.redirects)
+      end
+  end
+
+  def down
```

Comment:
Yeah I'd agree that since redirects isn't necessarily compatible with alternative path it might not be safe to do so. Presumably we should already have relatively good faith that the redirects collection works before this goes out too?